bookkeeper-server/src/test/java/org/apache/bookkeeper/client/TestGetBookieInfoTimeout.java

```diff
@@ -62,8 +65,17 @@ public TestGetBookieInfoTimeout() {
     @Before
     public void setUp() throws Exception {
         super.setUp();
-        channelFactory = new NioClientSocketChannelFactory(Executors.newCachedThreadPool(), Executors
-                .newCachedThreadPool());
+        if (SystemUtils.IS_OS_LINUX) {
```

Comment:
Actually, in this case it is redundant. The test is testing some other functionality and we should just use the portable `NioEventLoopGroup()`.